src/app/components/header/header.component.ts

```diff
@@ -11,16 +11,43 @@ import { selectUser } from '../../redux/selectors/user.selectors';
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.scss']
 })
-export class HeaderComponent implements OnInit{
+export class HeaderComponent implements OnInit {
   user$: Observable<User>;
+  showLogout: boolean = false;
+  showConfirmationModal: boolean = false;
+
+  constructor(private authService: AuthService, private userService: UserService) {}
 
-  constructor(private authService: AuthService, private userService: UserService, private store: Store){}
-  
   ngOnInit(): void {
-    this.user$ = this.store.pipe(select(selectUser));
+    this.user$ = this.userService.getUserInfoFromLocalStorage();
+  }
+
+  show() {
+    this.showLogout = !this.showLogout;
+  }
+
+  showConfirmation(){
+    this.showConfirmationModal = !this.showConfirmationModal;
   }
 
-  logout(){
+  close(){
```

Comment:
Please add a space after method call
```suggestion
  close() {
```